tools/lib/provision.py

```diff
@@ -430,7 +430,9 @@ def main(options):
         run_as_root(["chown", "%s:%s" % (user_id, user_id), NODE_MODULES_CACHE_PATH])
         setup_node_modules(prefer_offline=True)
     except subprocess.CalledProcessError:
-        print(WARNING + "`yarn install` failed; retrying..." + ENDC)
+        print(WARNING +
+              "`yarn install` failed, this is usually indicative of a network issue; retrying..." +
+              ENDC)
```

Comment:
I don't think this is the right place to give an error; we should do it if the second `setup_node_modules fails.

Reply:
Updated this with a sample output in #12699